doc/release.rst

```diff
@@ -121,6 +121,7 @@ Release notes for series before 1.10 are also available:
     release/policy
     release/legacy-policy
     release/release-calendar
+    release/compatibility
```

Comment:
This might need to change after https://github.com/tarantool/doc/pull/2743 is merged.